### DISCUSSION TITLE
Virtual text not working on javascript files

### DISCUSSION THREAD

**Discussion by gabrielrov** [hedyhli/outline.nvim#102] [Help]
## Description
On javascript files, virtual text does not appear, as if  `show_symbol_details` were disabled.

On lua:
![Screenshot_20241015_120718](https://github.com/user-attachments/assets/cda148bf-9e01-4a06-a1ba-e9ff402a8537)
On javacript:
![Screenshot_20241015_120829](https://github.com/user-attachments/assets/21d7a23d-c48f-4a71-8591-8b2a2f4ce189)

## Neovim setup

- Neovim version:   0.11.0
- Outline.nvim version:  6c44527

- Plugin manager:  Lazy.nvim

Config:
```
    require('outline').setup({
      outline_window = {
        position = 'right', -- left / right
        auto_close = false, -- Closes window if item is selected with enter
        focus_on_open = false,
      },
      outline_items = {
        show_symbol_details = true, -- Extra symbol details as virtual text
        show_symbol_lineno = false, -- Line numbers for each symbol
      },
      keymaps = {
        fold_all = { 'W', '<C-h>' },
        unfold_all = { 'E', '<C-l>' },
      },
    })


```
`:OutlineStatus` output:
```
 Filetype of current or attached buffer: javascript
 Symbols filter:
 
     (not configured)
 
 Default symbols filter:
 
     (all symbols included)
 
 Configured providers are: lsp, coc, markdown, norg.
 Current provider: lsp
 Provider info:
 
     client: ts_ls
 
 Outline window is open.
 Code window is active.

```

**Comment by hedyhli** [comment]
I have a feeling this might also be LSP dependent, but will have a look into it.

**Comment by davay** [comment]
Seems to be the case with Python too (pyright). I can see the virtual text in Lua too.
